src/styles/ForecastSummary.css

```diff
@@ -0,0 +1,37 @@
+.forecast-summary {
+  font-size: 25px;
+  padding: 15px;
+  background-color: rgba(240, 248, 255, 0.06);
+  border-inline: solid rgba(184, 180, 180, 0.523) 2px;
```

Comment:
The inline border with the radius is a very nice effect. I also like the transparency using the rgba color property